client/keys/utils.go

```diff
@@ -82,10 +75,33 @@ func ReadPassphraseFromStdin(name string) (string, error) {
 	return passphrase, nil
 }
 
-// initialize a keybase based on the configuration
+// TODO make keybase take a database not load from the directory
+
+// GetKeyBase initializes a keybase based on the configuration.
+func GetKeyBase() (keys.Keybase, error) {
+	rootDir := viper.GetString(cli.HomeFlag)
+	return GetKeyBaseFromDir(rootDir)
+}
+
+// GetKeyBaseWithWritePerm initialize a keybase based on the configuration with write permissions.
+func GetKeyBaseWithWritePerm() (keys.Keybase, error) {
+	rootDir := viper.GetString(cli.HomeFlag)
```

Comment:
Feel free to declare this out of scope of this PR, but it would be super nice to make this `rootDir` string passed in as a parameter instead of globally accessed.

Reply:
I like the API exposed here. It explicitly mentions the permissions you are getting when you open the database.

Reply:
The point I was making was removing the viper global variable, in favor of always explicitly passing in the rootDir, not about the permission itself.

Reply:
++

Reply:
Agreed, I guess it can be a separate PR though.

Reply:
@ValarDragon I agree, let's do it in a separate PR, shall we?